### ISSUE TITLE
Over-hyping

### ISSUE THREAD

**Issue by aggregate1166877** [nlpxucan/WizardLM#5]
Your README claims this model outperforms ChatGPT with "complex tasks." How should we define complex tasks? I asked it to write me shader code for Open GL ES 2.0, and it generated gibberish.

Once models fail a test like that, I like to ask them science questions because most of them will spew garbage (such as the Earth having 5 moons, or Jupiter being 10x larger than it actually is). I asked Wizard [via this interface](https://6f8173a3550ed441ab.gradio.live/) to describe the Moon... so it told me the Moon directly influences human menstrual cycles:

![1](https://user-images.githubusercontent.com/5172011/234557402-5aa5069e-4f89-4426-a257-610ff1807a72.jpg)

I then completely changed the parameters from the default, using temp 0.69, top p 0.5, top k 0, and it basically repeated itself:

![2](https://user-images.githubusercontent.com/5172011/234557425-6424f850-098e-453f-a0c3-9dee8bac4986.jpg)

There is no universe in which ChatGPT would say this.

_Please don't misunderstand_ - I'm by no means trying to degrade your work. It really does seem incredible, and I can't wait to see how this turns out. It kicks ass with some other tests. But claiming you've defeated the industry standard God only harms progress. The Alpaca team did the same and claimed they too had beaten ChatGPT. Please be honest in your README. Perhaps this model fulfilled your exact use case; but the goal of defeating the ChatGPT has not been achieved.

**Comment by nlpxucan** [comment]
> _Please don't misunderstand_ - I'm by no means trying to degrade your work. It really does seem incredible, and I can't wait to see how this turns out. It kicks ass with some other tests. But claiming you've defeated the industry standard God only harms progress. The Alpaca team did the same and claimed they too had beaten ChatGPT. Please be honest in your README. Perhaps this model fulfilled your exact use case; but the goal of defeating the ChatGPT has not been achieved.

Sincerely thank you for pointing out possible problems, We were also surprised when we received the human evaluation results, but decided to report everything truthfully. The reason is that our human evaluation results reveal that although ChatGPT is one of the world's strongest NLP models, it will not always perform perfectly in every case. It still has weaknesses and cannot perfectly handle every problem correctly. This is the message we want to convey to more NLP researchers and developers.

In both our paper and GitHub, we only claimed that we perform better than ChatGPT in the high-complexity part of our test set (Diffcult >=8). Furthermore, every time we made this claim, in order to eliminate any possible misunderstandings, we will immediately add a clarification that the overall quality of WizardLM is not as good as ChatGPT. 

The following are our respective claims in paper and github

In paper
"By analyzing the human evaluation results of the high complexity part, we demonstrate that outputs from our WizardLM model are preferred to outputs from OpenAI ChatGPT. Even though **WizardLM still lags behind ChatGPT** in some aspects"


In Github
"Although on our complexity-balanced test set, WizardLM-7B outperforms ChatGPT in the high-complexity instructions, **it still lag behind ChatGPT** on the entire test set" 

In summary, thank you very much for your kind discussion, this gives us a good opportunity to clarify again. We hope that we can always remain humble and make improving model performance as our common and unique goal.

**Comment by arsserpentarium** [comment]
wiz is worse, than chatgpt by closed domain qa. But it does what it told slightly better IMO. I think, LLM should know as less as possible facts. And should do ass complex as possible tasks. Sadly, it's problematic to add db usage to llama architecture.

**Comment by bkutasi** [comment]
I feel like this is the best model that is small enough and accurate enough to become widespread while being pretty accurate. Definitely better than oasst in the same parameter range. Of course in the grand scheme of things it's definitely not better than ChatGPT, but in some areas it could defeat it. We desperately need better comparison tools.

However, the problem is that we do not have standardized test for LMMs yet. I think we are currently missing crucial tests for claims like "x better than y", so the researches are stuck to their own devices to benchmark their models to other models.
There are some tests available currently:

https://github.com/stanford-crfm/helm
and
https://github.com/manyoso/haltt4llm
and
https://github.com/microsoft/AGIEval

**Comment by teknium1** [comment]
I find the model to be SOTA 7B model there is so, I don't agree that it is overhyped

**Comment by aggregate1166877** [comment]
> I find the model to be SOTA 7B model there is so, I don't agree that it is overhyped

Within context, my complaint was specifically claiming it outperforms ChatGPT. This is the second research effort (to my knowledge) to make such a claim incorrectly. When someone eventually does create something that beats ChatGPT, we run the risk that no one will give them any notice because "yeah right, the 17th model now that 'beats' ChatGPT."

I never said this model isn't good. In fact, I said the opposite. They didn't need to claim it's better than ChatGPT for people to appreciate its immense value. It's therefore over-hyped in that it makes (arguably) misleading claims, especially when it really didn't need to do so. They didn't need to coat it with unnecessary allure.

**Comment by CarlKenner** [comment]
Outperforming ChatGPT on complex tasks is a very important feature, which demonstrates the value of their training technique. They are right to hype it.

Unless the flaws you mention are caused by their approach, they are not relevant. The model's knowledge is limited by the training of the original Llama model it is based on, and by the much smaller number of parameters.

Remember that this is a *research* model. It isn't for consumer use. They are trying to contribute to AI research, and promoting the ways in which they have done so. I think they have moved AI research forward and their technique will probably be adopted by other models going forward.

**Comment by teknium1** [comment]
Random side note, somewhat relevant, https://www.youtube.com/watch?v=SaJ8wyKMBds

**Comment by aggregate1166877** [comment]
I personally feel the README has been updated to be fair and factual. I'm closing this task, please let me know if that's inaappropriate.
